site/specs-temp/numbers.json

```
@@ -1701,7 +1701,7 @@
     "/accounts/{accountId}/bulkPortins": {
       "get": {
         "tags": [
-          "Porting"
+          "Bulk port-in"
```

```suggestion
          "Bulk Port-In"
```
Tags should be `Title Case`

---

@maksymmateiuk can you repeat for all of the Bulk Port-In tags? 

looks like `I` was missed in last commit as well.